modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/util/ObjectEntryFieldNameUtil.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.rest.internal.util;
+
+import com.liferay.petra.string.StringBundler;
+
+/**
+ * @author Alberto Javier Moreno Lage
+ */
+public class ObjectEntryFieldNameUtil {
+
+	public static String getObjectRelationshipIdFieldName(
```

Comment:
I think this method should be in the Objects service class

Reply:
Maybe better to forget about this method. The PR is taking too long. We should move to the Objects package and inform/talk to Object's team to let them know